apex/contrib/test/optimizers/test_dist_adam.py

```diff
@@ -1,17 +1,13 @@
 from contextlib import contextmanager
 import io
-from typing import Optional, Tuple
+from typing import Callable, Optional, Tuple
 import unittest
 import warnings
 
 import torch
 from torch.testing._internal import common_utils
 
-SKIP_TEST = None
-try:
-    from apex.contrib.optimizers.distributed_fused_adam import DistributedFusedAdam
-except ImportError as e:
-    SKIP_TEST = e
+from apex.contrib.optimizers.distributed_fused_adam import DistributedFusedAdam
```

Comment:
I'm not sure why it's okay to remove the guard here off the top of my head?

Reply:
On second thought, it would be good to restore the guard in case Apex has been built without the distopt CUDA kernels.